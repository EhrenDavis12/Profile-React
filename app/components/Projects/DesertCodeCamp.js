import React from 'react';
import { P, Div, SectionBreak, Img } from 'components/SharedStyles/styles';

const DesertCodeCampIcon = require('images/DesertCodeCampIcon.png');

function DesertCodeCamp() {
  return (
    <div>
      <SectionBreak>
        <h4>Desert Code Camp Scheduler</h4>
        <P>
          React Native project Created for the purpose of favoring and
          scheduling sessions at the code camp.
        </P>
        <P>
          Desert Code Camp is a free 1 day event in Phoenix Arizona. This is a
          conference targeting the development community to inspire and educated
          at all levels.
        </P>
        <P>
          This project was the front end app for android phones. The Backend API
          already existed and published by the conference proprietor.
        </P>
      </SectionBreak>

      <div className="row">
        <div className="col-md-7">
          <Div>
            <div>Duration:</div>
            <ul>
              <li>One Week to learn React Native</li>
              <li>Three Weeks to program the project</li>
              <li>One Week for deployment to google play store</li>
              <li>No further updates will be made to this project</li>
            </ul>
          </Div>
          <Div>
            <div>
              You may see the code at the{' '}
              <a href="https://github.com/EhrenDavis12/DesertCodeCamp">
                GetHub Repository
              </a>
            </div>
            <div>
              You may download the project in the{' '}
              <a href="https://play.google.com/store/apps/details?id=com.ehrendavis.desertcodecampscheduler">
                Google Play Store
              </a>
            </div>
          </Div>
        </div>
        <div className="col-md-5 text-center">
          <Img src={DesertCodeCampIcon} alt="desertCodeCampIcon" />
        </div>
      </div>
    </div>
  );
}

export default DesertCodeCamp;
