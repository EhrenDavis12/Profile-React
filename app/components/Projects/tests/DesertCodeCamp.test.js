import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import DesertCodeCamp from '../DesertCodeCamp';

describe('<DesertCodeCamp />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<DesertCodeCamp />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
