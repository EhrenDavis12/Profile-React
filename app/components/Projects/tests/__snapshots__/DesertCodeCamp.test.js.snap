// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DesertCodeCamp /> should render the page 1`] = `
<div>
  <styles__SectionBreak>
    <h4>
      Desert Code Camp Scheduler
    </h4>
    <styles__P>
      React Native project Created for the purpose of favoring and scheduling sessions at the code camp.
    </styles__P>
    <styles__P>
      Desert Code Camp is a free 1 day event in Phoenix Arizona. This is a conference targeting the development community to inspire and educated at all levels.
    </styles__P>
    <styles__P>
      This project was the front end app for android phones. The Backend API already existed and published by the conference proprietor.
    </styles__P>
  </styles__SectionBreak>
  <div
    className="row"
  >
    <div
      className="col-md-7"
    >
      <styles__Div>
        <div>
          Duration:
        </div>
        <ul>
          <li>
            One Week to learn React Native
          </li>
          <li>
            Three Weeks to program the project
          </li>
          <li>
            One Week for deployment to google play store
          </li>
          <li>
            No further updates will be made to this project
          </li>
        </ul>
      </styles__Div>
      <styles__Div>
        <div>
          You may see the code at the
           
          <a
            href="https://github.com/EhrenDavis12/DesertCodeCamp"
          >
            GetHub Repository
          </a>
        </div>
        <div>
          You may download the project in the
           
          <a
            href="https://play.google.com/store/apps/details?id=com.ehrendavis.desertcodecampscheduler"
          >
            Google Play Store
          </a>
        </div>
      </styles__Div>
    </div>
    <div
      className="col-md-5 text-center"
    >
      <styles__Img
        alt="desertCodeCampIcon"
        src="IMAGE_MOCK"
      />
    </div>
  </div>
</div>
`;
